drivers/cmd/webhook/main_test.go

```diff
@@ -0,0 +1,58 @@
+// Copyright 2019 Honey Science Corporation
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, you can obtain one at http://mozilla.org/MPL/2.0/.
+
+package main
+
+import (
+	"context"
+	"net/http"
+	"os"
+	"testing"
+
+	"github.com/honeydipper/honeydipper/internal/daemon"
+	"github.com/honeydipper/honeydipper/pkg/dipper"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestMain(m *testing.M) {
+	if dipper.Logger == nil {
+		logFile, err := os.Create("test.log")
+		if err != nil {
+			panic(err)
+		}
+		defer logFile.Close()
+		dipper.GetLogger("test", "INFO", logFile, logFile)
+	}
+	driver = &dipper.Driver{Service: "test"}
+	log = driver.GetLogger()
+	m.Run()
+}
+
+func TestExtractEvent(t *testing.T) {
+	var eventData map[string]interface{}
+	var server *http.Server
+	hookHandlerTest := func(w http.ResponseWriter, r *http.Request) {
+		eventData = extractEventData(w, r)
+		w.WriteHeader(http.StatusOK)
+		go server.Shutdown(context.Background())
+	}
+	server = &http.Server{
+		Addr:    "127.0.0.1:8999",
+		Handler: http.HandlerFunc(hookHandlerTest),
+	}
+	daemon.Children.Add(1)
+	go func() {
+		defer daemon.Children.Done()
+		server.ListenAndServe()
+	}()
+	http.Get("http://127.0.0.1:8999")
```

Comment:
```suggestion
	resp, _  := http.Get("http://127.0.0.1:8999")
	resp.Body.Close()
```

From go net/http documents at https://golang.org/pkg/net/http/#Client

> The client must close the response body when finished with it: